lua/nvim-treesitter/locals.lua

```diff
@@ -172,6 +174,40 @@ M.get_definitions_lookup_table = ts_utils.memoize_by_buf_tick(function(bufnr)
   return result
 end)
 
+--- Gets all the scopes of a definition based on the scope type
+-- Scope types can be
+--
+-- "parent": Uses the parent of the containing scope, basically, skipping a scope
+-- "global": Uses the top most scope
+-- nil: Uses the containg scope of the definition
```

Comment:
I think we may need to have `local` as an option too. This is, nil explicitly (we don't have something like unset or set to nil I guess).

Reply:
This would be useful to "reset" the value in another query.

Reply:
done :+1: